src/content/dao/index.md

```diff
@@ -69,6 +70,17 @@ _Typically used for more closer-knit, human-centric organizations like charities
 
 [MolochDAO](http://molochdao.com/) – MolochDAO is focused on funding Ethereum projects. They require a proposal for membership so the group can assess whether you have the necessary expertise and capital to make informed judgments about potential grantees. You can't just buy access to the DAO on the open market.
 
+### Reputation-based membership {#reputation-based-membership}
+
+Reputation represents ‘proof-of-work’ and grants voting power, but unlike shares - no ownership, and members do not have a claim on the treasury. Unlike liquid tokens, reputation cannot be bought, sold, transferred or delegated - it is only earned, but it can also be slashed. Reputation-based DAOs do not require reputation or tokens for someone to submit a proposal for on-chain voting, it is permissionless. Prospective members can freely submit proposals to join the DAO, and in exchange for their contributions request to receive reputation and tokens as a reward.
+Due to their unique nature, wich doesn't share the constraints of token governance, reputation-based systems allow DAOs to expand to multiple networks and conduct cross-chain governance, without relying on bridges. That paired with holographic consensus also allows for DAOs to scale without compromising on resiliency.
+
+_Typical for decentralized development and governance of protocols and dApps, but also well suited to a diverse set of organizations like charities, worker collectives, investment clubs, etc._
+
+#### A famous example {#reputation-example}
+
+[DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective that has been building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and mananage funds trustlessly. It's not possible for anyone to buy they way into influencing its future.
```

Comment:
```suggestion
[DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and manage funds, meaning no one can buy their way into influencing its future.
```